arangod/RocksDBEngine/RocksDBCuckooIndexEstimator.h

```diff
@@ -337,14 +337,14 @@ class RocksDBCuckooIndexEstimator {
 
   double computeEstimate() {
     READ_LOCKER(locker, _lock);
-    if (0 == (_nrTotal + _nrCuckood)) {
-      TRI_ASSERT(0 == _nrTotal);
-      TRI_ASSERT(0 == _nrCuckood);
+    if (0 == _nrTotal) {
+      TRI_ASSERT(0 == _nrUsed);
       // If we do not have any documents we have a rather constant estimate.
       return 1.0;
     }
-    return (static_cast<double>(_nrUsed) /
-            static_cast<double>(_nrTotal + _nrCuckood));
+    TRI_ASSERT(_nrUsed <= _nrTotal);
+
+    return (static_cast<double>(_nrUsed) / static_cast<double>(_nrTotal));
```

Comment:
Why did you kick out `_nrCuckood`?
This was used as a smoothing factor for large collections with many different values (more than this estimate could handle)
As a test try to insert 10.000.000 distinct values into this index. => Estimate should still be 1.0

Reply:
As discussed elsewhere (and included now here to preserve for the record), prior to very recent changes, `_nrCuckood` was not used in the computation. It had been decided several months ago that the logic to use `_nrCuckood` as a smoothing factor is fairly complicated, and seems to bias the estimate one way or another so that it helps in some situations and hurts in others. It was unclear whether this is useful or not, so at least for now we will leave it in the same state as matches our public releases for stability.